### DISCUSSION TITLE
How to determine the condition, prohibit the execution of subsequent code, and immediately jump?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#27476] [General]
How to determine the condition, prohibit the execution of subsequent code, and immediately jump?

![image](https://github.com/nuxt/nuxt/assets/26525599/870cdf7a-81c3-4eb9-8d21-91c2f04faa4d)

**Comment by xjccc** [comment]
```
return navigateTo('/')
```

**Reply by bbhxwl** [reply]
Thank you, what should I do if I don't execute the following code?

**Reply by bbhxwl** [reply]
> ```
> return navigateTo('/')
> ```

Are you right?


![image](https://github.com/nuxt/nuxt/assets/26525599/ac850ddc-e214-44c4-85f1-04327f8457fe)

**Reply by xjccc** [reply]
oh ... I know that.

```
await navigateTo('/')

console.log(1)
```

Its seems not work ...

```
await navigateTo('/', { redirectCode: 301 })

console.log(1)
```

When i add `redirectCode`, its seems return the next code?
